pkg/spec/gitops.go

```diff
@@ -533,6 +549,41 @@ func parsePolicies(top map[string]json.RawMessage, result *GitOps, baseDir strin
 	return multiError
 }
 
+func parsePolicyRunScript(baseDir string, teamName *string, policy *Policy, scripts []BaseItem) error {
+	if policy.RunScript == nil {
+		policy.ScriptID = ptr.Uint(0) // unset the script
+		return nil
+	}
+	if policy.RunScript != nil && policy.RunScript.Path != "" && teamName == nil {
+		return errors.New("run_script can only be set on team policies")
+	}
+
+	if policy.RunScript.Path == "" {
+		return errors.New("empty run_script path")
+	}
+
+	_, err := os.Open(resolveApplyRelativePath(baseDir, policy.RunScript.Path))
```

Comment:
might have a leak here since we're not capturing and closing.  may want to use `os.Stat` instead

Reply:
Fixed in [ce80b1d](https://github.com/fleetdm/fleet/pull/22472/commits/ce80b1d63e670b8c3a31ff1c089999be20667283)